spec/feature/feature_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'capybara/rspec'
+require 'order'
+
+feature 'A customer wants to order takeaway' do
+  scenario 'Customer is able to view list of dishes from the menu' do
+  	order = Order.new
+  	expect(order.print_menu).to be_a_kind_of(Hash)
```

Comment:
Avoid tests that do type-checking. The reason for this is that if, in future, you change the implementation of Order such that the menu is no longer a hash, but it still does the same job, this test will break.